code/modules/religion/religion_sects.dm

```diff
@@ -213,7 +223,141 @@
+	from your defeated shell."
+	//a list for keeping track of how burdened each member is
+
+/datum/religion_sect/burden/on_conversion(mob/living/carbon/new_convert)
+	if(!ishuman(new_convert))
+		to_chat("<span class='warning'>[GLOB.deity] needs higher level creatures to fully comprehend the suffering. You are not burdened.</span>")
+	else
+		var/datum/dna/holy_dna = new_convert.dna
+		holy_dna.add_mutation(/datum/mutation/human/burdened)
+
+/datum/religion_sect/burden/tool_examine(mob/living/carbon/human/burdened) //display money policy
+	var/datum/mutation/human/burdened/burdenmut = burdened.dna.check_mutation(/datum/mutation/human/burdened)
+	if(burdenmut)
+		return "<span class='notice'>You are at burden level [burdenmut.burden_level]/6.</span>"
+	return "<span class='notice'>You are not burdened.</span>"
+
+#define MINIMUM_YUCK_REQUIRED 5
+
+/datum/religion_sect/maintenance
+	name = "Maintenance God"
+	desc = "A sect based around the maintenance shafts of the station."
+	altar_icon_state = "convertaltar-maint"
+	alignment = ALIGNMENT_EVIL //while maint is more neutral in my eyes, the flavor of it kinda pertains to rotting and becoming corrupted by the maints
+	convert_opener = "\"Your kingdom in the darkness.\"<br>\
+	Sacrifice the organic slurry created from rats dipped in welding fuel to gain favor. Exchange favor to adapt to the maintenance shafts."
+	rites_list = list(/datum/religion_rites/maint_adaptation, /datum/religion_rites/adapted_food, /datum/religion_rites/ritual_totem)
+	desired_items = list(/obj/item/reagent_containers)
+
+/datum/religion_sect/maintenance/sect_bless(mob/living/blessed_living, mob/living/user)
+	if(!ishuman(blessed_living))
+		return TRUE
+	var/mob/living/carbon/human/blessed = blessed_living
+	if(blessed.reagents.has_reagent(/datum/reagent/drug/maint/sludge))
+		to_chat(blessed, "<span class='warning'>[GLOB.deity] has already empowered them.</span>")
+		return TRUE
+	blessed.reagents.add_reagent(/datum/reagent/drug/maint/sludge, 5)
+	blessed.visible_message("<span class='notice'>[user] empowers [blessed] with the power of [GLOB.deity]!</span>")
+	to_chat(blessed, "<span class='boldnotice'>The power of [GLOB.deity] has made you harder to wound for awhile!</span>")
+	playsound(user, "punch", 25, TRUE, -1)
+	SEND_SIGNAL(blessed, COMSIG_ADD_MOOD_EVENT, "blessing", /datum/mood_event/blessing)
+	return TRUE //trust me, you'll be feeling the pain from the maint drugs all well enough
+
+/datum/religion_sect/maintenance/on_sacrifice(obj/item/reagent_containers/offering, mob/living/user)
+	if(!istype(offering))
+		return
+	var/datum/reagent/yuck/wanted_yuck = offering.reagents.has_reagent(/datum/reagent/yuck, MINIMUM_YUCK_REQUIRED)
+	var/favor_earned = offering.reagents.get_reagent_amount(/datum/reagent/yuck)
+	if(!wanted_yuck)
+		to_chat(user, "<span class='warning'>[offering] does not have enough Organic Slurry for [GLOB.deity] to enjoy.</span>")
```

Comment:
```suggestion
		to_chat(user, "<span class='warning'>[offering] does not have enough organic slurry for [GLOB.deity] to enjoy.</span>")
```
idk if not capitalizing this makes it too unclear that we're looking for a specific chem here instead of just nutriment or whatever, though